File: Lab_1/Parte_I.py
def main():
    while True:
        # Menú de opciones
        print("1 - Area de un cuadrado")
        print("2 - Area de un rectángulo")
        print("3 - Perímetro de un cuadrado")
        print("4 - Perimetro de un rectángulo")
        print("5 - Area de un triangulo rectángulo")
        print("6 - Salir")
        opcion = int(input("¿Que formula deseas ingresar? "))
       
        if opcion == 6:
            print("Saliendo...")
            break
       
        # Variables para las dimensiones
        base = 0
        altura = 0
       
        if opcion == 1:
            base = float(input("Introduce la longitud del lado del cuadrado: "))
            areaCuadrado = base * base
            print(f"Area del cuadrado: {areaCuadrado * 100} cm²")
       
        elif opcion == 2:
            base = float(input("Introduce la base del rectangulo: "))
            altura = float(input("Introduce la altura del rectangulo: "))
            areaRectangulo = base * altura
            print(f"Area del rectangulo: {areaRectangulo * 100} cm²")
       
        elif opcion == 3:
            base = float(input("Introduce la longitud del lado del cuadrado: "))
            perimetroCuadrado = base * 4
            print(f"Perimetro del cuadrado: {perimetroCuadrado * 100} cm")
       
        elif opcion == 4:
            base = float(input("Introduce la base del rectangulo: "))
            altura = float(input("Introduce la altura del rectangulo: "))
            perimetroRectangulo = (base * 2) + (altura * 2)
            print(f"Perimetro del rectangulo: {perimetroRectangulo * 100} cm")
       
        elif opcion == 5:
            base = float(input("Introduce la base del triangulo: "))
            altura = float(input("Introduce la altura del triangulo: "))
            areaTriangulo = (base * altura) / 2
            print(f"Area del triangulo: {areaTriangulo * 100} cm²")
       
        else:
            print("Opcion invalida, por favor elige una opción entre 1 y 6.")

if __name__ == "__main__":
    main()

File: Lab_1/Parte_I.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package parte1;

import java.util.Scanner;

/**
 *
 * @author escom
 */
public class Parte1 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int opcion;
        
        do {
            // Menú de opciones
            System.out.println("1 - Area de un cuadrado");
            System.out.println("2 - Area de un rectangulo");
            System.out.println("3 - Perimetro de un cuadrado");
            System.out.println("4 - Perimetro de un rectángulo");
            System.out.println("5 - Area de un triangulo rectangulo");
            System.out.println("6 - Salir");
            System.out.print("¿Que formula deseas ingresar? ");
            
            // Leer la opción del usuario
            opcion = scanner.nextInt();
            
            // Variables para las dimensiones
            float base = 0, altura = 0;
            
            switch (opcion) {
                case 1:
                    System.out.print("Introduce la longitud del lado del cuadrado: ");
                    base = scanner.nextFloat();
                    float areaCuadrado = base * base;
                    System.out.println("Area del cuadrado: " + areaCuadrado*100 + "cm²");
                    break;
                case 2:
                    System.out.print("Introduce la base del rectangulo: ");
                    base = scanner.nextFloat();
                    System.out.print("Introduce la altura del rectangulo: ");
                    altura = scanner.nextFloat();
                    float areaRectangulo = base * altura;
                    System.out.println("Area del rectangulo: " + areaRectangulo*100 + "cm²");
                    break;
                case 3:
                    System.out.print("Introduce la longitud del lado del cuadrado: ");
                    base = scanner.nextFloat();
                    float perimetroCuadrado = base * 4;
                    System.out.println("Perimetro del cuadrado: " + perimetroCuadrado*100 + "cm");
                    break;
                case 4:
                    System.out.print("Introduce la base del rectangulo: ");
                    base = scanner.nextFloat();
                    System.out.print("Introduce la altura del rectangulo: ");
                    altura = scanner.nextFloat();
                    float perimetroRectangulo = (base * 2) + (altura * 2);
                    System.out.println("Perimetro del rectangulo: " + perimetroRectangulo*100 + "cm");
                    break;
                case 5:
                    System.out.print("Introduce la base del triangulo: ");
                    base = scanner.nextFloat();
                    System.out.print("Introduce la altura del triangulo: ");
                    altura = scanner.nextFloat();
                    float areaTriangulo = (base * altura) / 2;
                    System.out.println("Area del triangulo: " + areaTriangulo*100 + "cm²");
                    break;
                case 6:
                    System.out.println("Saliendo...");
                    break;
                default:
                    System.out.println("Opción inválida, por favor elige una opción entre 1 y 6.");
                    break;
            }
            
        } while (opcion != 6);

        
    }
    
}
